packages/wrangler/src/__tests__/d1/execute.test.ts

```diff
@@ -18,7 +18,7 @@ describe("execute", () => {
 		});
 
 		await expect(
-			runWrangler("d1 execute --command 'select 1;'")
+			runWrangler("d1 execute db --command 'select 1;'")
```

Comment:
This test forgot to specify the `database` positional argument. It was passing before because the `command` argument was being split by space, resulting in command = `'select ` and `database` = `1;'`